Problem1/src/UnitTest.java

```diff
@@ -0,0 +1,19 @@
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+
+public class UnitTest {
+
+	@Test
+	public void testAllUnique1 (){
```

Comment:
good, one important rule of designing unit tests is to cover edge cases like ampty strings etc. You have to make work every route inside your ``checkAllUniqueChars1```
One important note. Instead of using things like checkAllUniqueChars1 ,checkAllUniqueChars2 - pelase add common itnerface and just provide two implementations.